app/services/service_provider_updater.rb

```diff
@@ -12,5 +18,11 @@ def idp_url
     def token_header
       { 'X-LOGIN-DASHBOARD-TOKEN' => Figaro.env.dashboard_api_token }
     end
+
+    def handle_error(status)
+      ::NewRelic::Agent.notice_error("ServiceProviderUpdater failed with "\
```

Comment:
My understanding is that `notice_error` takes an error, not just a string.  I can run with this since I just did similar on the push notification failures.